### ISSUE TITLE
Better document this plugin

### ISSUE THREAD

**Issue by jorisw** [floatinghotpot/cordova-plugin-wifi#3]
This plug-in seems to work at least for retrieving WiFi status, but I don't understand the documentation very well.

Please fully document all the methods and their parameters.

Specifically, what are the `istrue` arguments meant for?

**Comment by msudol** [comment]
agree, might be a useful plugin if it were documented.. don't even know what the getwifiinfo callback object has in it without inspecting it or looking at the source code.
